packages/table/components/Column.tsx

```diff
@@ -18,7 +18,15 @@ export interface ColumnProps {
   /**
    * width should return a column width in pixels.
    */
-  width: (args: WidthArgs) => number;
+  width?: (args: WidthArgs) => number;
```

Comment:
regarding `minWidth` and `maxWidth`, what do you think about passing in `caluclatedWith` with `args` here instead and let the function do it? It feels a bit more streamlined if one function does all instead of having a function and 2 config thingies 🤔

Reply:
That's possible. I felt like it was more straightforward to have them as separate props, but I'm curious to hear what other reviewers think.

Happy to make that change if even one other person thinks your suggestion is a better developer experience :)